lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -305,9 +305,11 @@ def self.resolve_automation_object(uri, user_obj, attr = nil, options = {}, read
 
   def self.ae_user_object(options = {})
     raise "user_id not specified in Automation request" if options[:user_id].blank?
-    # raise "group_id not specified in Automation request" if options[:miq_group_id].blank?
+    raise "miq_group_id not specified in Automation request" if options[:miq_group_id].blank?
```

Comment:
Ok, please like the related PR back here when it is ready, I would like to have it available before merging this one since we know it will break Automation requests.